### PR TITLE
Adding latest datamover config changes

### LINKED ISSUE #13: Sync Latest DataMover Configs

### PR DESCRIPTION
**Pull request by nsarras**

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -172,13 +172,15 @@
...
 #       - /mnt/nfs_share/15gb-B
+# datamover_nfs_port: 2049
 # datamover_bucket: bucket
 # datamover_client_lib: dss_client
 # datamover_logging_path: /var/log/dss
 # datamover_logging_level: INFO
 # datamover_gcc_version: 5.1
 # datamover_index_data_queue_size: 50000
 # datamover_awslib_log_debug: 0
+# datamover_compaction: yes
```
Comment (velomatt) [inline group_vars/all.yml]: This var is already set: https://github.com/OpenMPDK/dss-ansible/blob/aece12f7f5b3f5f0fc1fe65e583494c31e7b9f59/group_vars/all.yml#L150

This is also adding another flavor of "yaml truthiness" but `yes` essentially has the same value as `true` in this case.

File: roles/create_datamover_conf/defaults/main.yml [hunk 2]
```diff
@@ -63,13 +63,15 @@ datamover_nfs_shares:
...
       - /mnt/nfs_share/15gb-B
+datamover_nfs_port: 2049
 datamover_bucket: bucket
 datamover_client_lib: dss_client
 datamover_logging_level: INFO
 datamover_gcc_version: 5.1
 datamover_index_data_queue_size: 50000
 datamover_awslib_log_debug: 0
 datamover_fs_mounted: false
+datamover_compaction: yes
```
Comment (velomatt) [inline roles/create_datamover_conf/defaults/main.yml]: Same as above comment

File: roles/create_datamover_conf/templates/config.json.j2 [hunk 3]
```diff
@@ -67,5 +67,6 @@
   },
   "aws": {
    "awslib_log_debug": {{ datamover_awslib_log_debug }}
-  }
+  },
+  "compaction": "{{ datamover_compaction }}"
```
Comment (velomatt) [inline roles/create_datamover_conf/templates/config.json.j2]: If `datamover_compaction` is `yes`, then the template will set this value to `true`.

see: https://zzz.buzz/notes/ansible-truth-table/

If we want it to print the string `yes` - we will need to do so explicitly. But if we set `datamover_compaction` to a string, then we can't use it to evaluate as a boolean in Ansible any more.

**Review by velomatt — CHANGES_REQUESTED** [review]
Please review inline comments

File: roles/create_datamover_conf/templates/config.json.j2 (hunk 3 above)
Comment (nsarras) [reply roles/create_datamover_conf/templates/config.json.j2]: ok, then I think we want to explicitly list "yes" as that is the underlying input that the DataMover code expects

**Review by velomatt — APPROVED** [review]
I have tested this and it works. Approved

**Pull request merged**